src/runner.py

```diff
@@ -155,16 +156,31 @@ def remove(self, remove_token: str) -> None:
                     raise RunnerRemoveError(f"Unable to remove {self.config.name}") from err
 
         # The runner should cleanup itself.  Cleanup on GitHub in case of runner cleanup error.
-        if isinstance(self.config.path, GitHubRepo):
-            self._clients.github.actions.delete_self_hosted_runner_from_repo(
-                owner=self.config.path.owner,
-                repo=self.config.path.repo,
-                runner_id=self.config.name,
-            )
-        if isinstance(self.config.path, GitHubOrg):
-            self._clients.github.actions.delete_self_hosted_runner_from_org(
-                org=self.config.path.org, runner_id=self.config.name
-            )
+        if self.status.runner_id is not None:
```

Comment:
Minor nitpick,
Would this
'''Python
if self.status.runner_id is None:
    Return
if isinstance(
...
'''
Help reduce indentation?

Reply:
Will be including this change in a future PR.